docs/home/HOWTO.md

```diff
@@ -70,6 +70,7 @@
   - [How to enable/disable Epiphany service user](./howto/SECURITY.md#how-to-enabledisable-epiphany-service-user)
   - [How to use TLS/SSL certificate with HA Proxy](./howto/SECURITY.md#how-to-use-tlsssl-certificate-with-ha-proxy)
   - [How to use TLS/SSL certificate with RabbitMQ](./howto/SECURITY.md#how-to-use-tlsssl-certificate-with-rabbitmq)
+  - [How to use TLS/SSL certificate with Kibana](./howto/SECURITY.md#how-to-use-tlsssl-certificate-with-kibana)
```

Comment:
Why is it here? Seems to be related to another PR.

Reply:
There is something with history after rebase. Too long to rework right now I think.